spring-cloud-dataflow-app-launcher/src/main/java/org/springframework/cloud/dataflow/app/resolver/AetherModuleResolver.java

```diff
@@ -193,6 +206,12 @@ private DefaultRepositorySystemSession newRepositorySystemSession(RepositorySyst
 		LocalRepository localRepo = new LocalRepository(localRepoPath);
 		session.setLocalRepositoryManager(system.newLocalRepositoryManager(session, localRepo));
 		session.setOffline(this.offline);
+		if (this.mavenProperties != null && this.mavenProperties.getConnectTimeout() > 0) {
+			session.setConfigProperty("aether.connector.connectTimeout", this.mavenProperties.getConnectTimeout());
```

Comment:
should use the constants in Aether's `ConfigurationProperties` instead of the String keys